ebmdatalab/maps.py

```
@@ -1,131 +1,142 @@
+        :return: Figure
+        """
+
         subplot_spec = gridspec.GridSpec(1, 1)[0]
-    if separate_london:
-        if london_layout == "horizontal":
-            gs = gridspec.GridSpecFromSubplotSpec(
-                nrows=1, ncols=2, width_ratios=[1, 2], subplot_spec=subplot_spec
-            )
-            ldn_ax = fig.add_subplot(gs[0, 0])
-            roe_ax = fig.add_subplot(gs[0, 1])
+        lon_data = self.final_df[self.final_df["is_london"] == True]
+        roe_data = self.final_df[self.final_df["is_london"] == False]
+        gs = gridspec.GridSpecFromSubplotSpec(nrows=1, ncols=2, width_ratios=[1, 2], subplot_spec=subplot_spec)
+        fig = plt.figure(figsize=size)
+        ldn_ax = fig.add_subplot(gs[0, 0])
+        ldn_ax.axis("off")
+        ldn_ax.set_title("London:\n{}".format(title), size=14)
+        roe_ax = fig.add_subplot(gs[0, 1])
+        if fit_legend == True:
+            divider = make_axes_locatable(roe_ax)
+            cax = divider.append_axes("right", size="5%", pad=0.1)
+        else:
+            cax = None
+        roe_ax.axis("off")
+        roe_ax.set_title("England (excluding London):\n{}".format(title))
+        roe_data.plot(column=plot_value, legend=legend, ax=roe_ax, cmap=cmap, cax=cax, edgecolor=edgecolor,
+                      linewidth=linewidth)
+        lon_data.plot(column=plot_value, legend=False, ax=ldn_ax, cmap=cmap, edgecolor=edgecolor,
+                      linewidth=linewidth)
+        if save == True:
+            self.plots[description] = fig
+        return fig
+
+    def create_map(self, plot_value, title,
+                            description=None,
+                            size=(12, 10),
+                            cmap="OrRd",
+                            legend=True,
+                            fit_legend=False,
+                            edgecolor="black",
+                            linewidth=0.1,
+                            save=False,
+                            **kwargs):
```

It doesn't look like `kwargs` is used in this method so it's better not to accept it otherwise incorrect keyword arguments get silently swallowed rather than triggering an error.